packages/fxa-settings/src/pages/Signup/container.test.tsx

```diff
@@ -154,6 +154,7 @@ function mockApolloClientModule() {
       loading: false,
       called: true,
       client: {} as ApolloClient<any>,
+      reset: () => {},
```

Comment:
@dschom You [were right](https://github.com/mozilla/fxa/pull/16105#discussion_r1401359495) that I needed to yarn install, I was using node modules from this branch heh.